### DISCUSSION TITLE
How to prevent cache

### DISCUSSION THREAD

**Discussion by bayaderpack** [a-h/templ#876] [Q&A]
I'm on Windows 11
I'm using this command to watch changes
```
templ generate --watch --proxy="http://localhost:8000" --cmd="go run ."
```
I'm also using tailwindcss sometimes if I use some for example new color lets say `bg-pink-400` I must actually go in browser and press CTRL+F5 to it shows.
I guess it because cache I'm not sure but can I somehow prevent cache or use any other solution?

**Comment by a-h** [comment]
I'm not sure you've provided enough context here for anyone to answer this question.

I'm guessing you have another command running in the background that updates a tailwind CSS bundle? Once that's run you need to trigger the templ proxy to reload, because the templ proxy only knows about changes to the templ file: https://templ.guide/commands-and-tools/live-reload-with-other-tools/#example

Specifically:

> `templ generate --watch` sends the event whenever a ".templ" file changes.
>
> Manually trigger it by sending a HTTP POST request to /_templ/reload/event endpoint. The templ CLI provides this via `templ generate --notify-proxy`.

If it's some other problem, you'll need to provide a reproduction.
